internal/fsutil/create.go

```diff
@@ -9,14 +9,23 @@ import (
 )
 
 type CreateOptions struct {
-	Path string
-	Mode fs.FileMode
-	Data io.Reader
-	Link string
+	Path        string
+	Mode        fs.FileMode
+	Data        io.Reader
+	Link        string
+	MakeParents bool
 }
 
+// Creates file according to passed CreateOptions.
```

Comment:
```
// Creates a filesystem entry according to the provided options.
...
```
(not just files, but directories, symlinks, etc)

Reply:
Done